cypress/e2e/wip/Tradershub/addSiblingsAccounts.cy.js

```diff
@@ -0,0 +1,64 @@
+import "@testing-library/cypress/add-commands"
+import {generateEpoch} from '../../../support/tradersHub'
+
+function createSiblingacct(i){
+    cy.findByTestId('dt_currency-switcher__arrow').click()
+    cy.findByRole('button', { name: 'Add or manage account' }).click()
+    cy.get('.currency-list__item').eq(i).click()
+    cy.findByRole('button', { name: 'Add account' }).click()
+    cy.findByRole('heading', { name: 'Success!' }).should('be.visible')
+    cy.findByRole('button', { name: 'Deposit now' }).should('be.enabled')
+    cy.findByRole('button', { name: 'Maybe later' }).click()  
+}
+function getCurrencyList() {
+  return cy.get('div.currency-list__items').find('label').then(($labels) => { 
+  const labelCount = $labels.length
+  return labelCount
+  })
+}
+describe("QATEST-5797, QATEST-5820 - Add siblings accounts", () => {
+    const epoch = generateEpoch()
+    const sign_up_mail = `sanity${epoch}@deriv.com`
```

Comment:
this should be camelCase as
```suggestion
    const signUpMail = `sanity${epoch}@deriv.com`
```